### PR TITLE
Feature/inscription form

### PR DESCRIPTION
**Pull request by Alexandra-Rdrgs**
Formulaire d'inscription sans connexion au back.

### CODE REVIEW

**Review by nibor51 — CHANGES_REQUESTED** [review]
Tty to use a variable for repeating classNames ["text-red-600 normal-case text-xs font-normal italic"]
it will be easier to maintain

File: frontend/src/components/InscriptionForm.jsx [hunk 1]
```diff
@@ -0,0 +1,358 @@
...
+            role="alert"
+          >
+            <span className="block sm:inline">
+              Votre inscription a été prise en compte.
+            </span>
+          </div>
+        )}
+        <div className="flex flex-wrap -mx-3 mb-6">
+          <div className="w-full md:w-1/2 px-3 mb-6 md:mb-0">
+            <label className="form-label" htmlFor="grid-firstname">
```
Comment (rocambille) [inline frontend/src/components/InscriptionForm.jsx:48]: try to merge the div and the label

**Review by rocambille — CHANGES_REQUESTED** [review]

**Comment by Alexandra-Rdrgs** [comment]
The changes requested have been made

File: frontend/src/components/InscriptionForm.jsx [hunk 2]
```diff
@@ -0,0 +1,321 @@
...
+      </h2>
+      {isSubmitSuccessful && (
+        <div
+          className="bg-lime-100 border border-lime-400 text-lime-700 mb-6 px-4 py-3 rounded"
+          role="alert"
+        >
+          <span className="block sm:inline">
+            Votre inscription a été prise en compte.
+          </span>
+        </div>
```
Comment (rocambille) [inline frontend/src/components/InscriptionForm.jsx:43]: another p to replace div+span ?

File: frontend/src/components/InscriptionForm.jsx [hunk 3]
```diff
@@ -0,0 +1,321 @@
...
+          {errors?.lastname?.type === "maxLength" && (
+            <p className="error-handler">
+              Votre prénom ne peut pas exceder 50 caractères.
+            </p>
+          )}
+          {errors?.lastname?.type === "pattern" && (
+            <p className="error-handler">
+              Ce champ n'accèpte pas la saisie de caractères spéciaux.
+            </p>
+          )}
```
Comment (rocambille) [inline frontend/src/components/InscriptionForm.jsx:110]: p are not allowed in labels : put them outside

**Pull request merged**